docs/user/00-20-configure-serverless.md

```diff
@@ -44,24 +44,56 @@ Follow these steps to use the external Docker registry in Serverless:
    ```bash
    kubectl create secret generic my-registry-config \
        --namespace kyma-system \
-       --from-literal=username={your-docker-reg-username} \
-       --from-literal=password={your-docker-reg-password} \
-       --from-literal=serverAddress={your-docker-reg-server-url} \
-       --from-literal=registryAddress={your-docker-reg-registry-url}
+       --from-literal=username={USERNAME} \
+       --from-literal=password={PASSWORD} \
+       --from-literal=serverAddress={SERVER_URL} \
+       --from-literal=registryAddress={REGISTRY_URL}
    ```
 
 >**TIP:** In case of DockerHub, usually the Docker registry address is the same as the account name.
 
-Example:
+Examples:
 
+<!-- tabs:start -->
+
+### **Docker Hub**
+
+   ```bash
+      kubectl create secret generic my-registry-config \
+          --namespace kyma-system \
+          --from-literal=username=kyma-rocks \
+          --from-literal=password=admin123 \
+          --from-literal=serverAddress=https://index.docker.io/v1/ \
+          --from-literal=registryAddress=kyma-rocks
```

Comment:
I would suggest keeping consistent way how we show example values for all fields

```suggestion
          --namespace kyma-system \
          --from-literal=username={USERNAME} \
          --from-literal=password={PASSWORD} \
          --from-literal=serverAddress=https://index.docker.io/v1/ \
          --from-literal=registryAddress={USERNAME}
```

or replace values for GCP and ACR with examples values